packages/web-components/src/button/button.ts

```diff
@@ -515,6 +424,99 @@ export class Button extends FASTElement {
   }
 }
 
+export class Button extends BaseButton {
```

Comment:
This class should have a documentation block to describe its purpose and to differentiate between it and the base class, and to mark it as `@public`.

Reply:
FYI we do not do this in anchor button

Reply:
We likely should given we're intending to make the API's available.

Reply:
Yup just pointing out other places where we aren't doing this handling those cases now.